### DISCUSSION TITLE
Use own certificate for manager

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#11916] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

512GB

### Storage for /

15TB

### Storage for /nsm

30TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hi,

how can I use our company ssl cert? I copied it on the manager, rename the old managerssl.* and copied ours with the same names, restart so-nginx and reboot. But only see the old ssl cert in browser.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment, marked as answer]
Have you tried following the documentation?
https://docs.securityonion.net/en/2.4/nginx.html#replacing-default-cert

**Comment by ByteAndBits** [comment]
just I say RTFM (read the fu**** manual).

Thank´s
